lib/realtime/tenants/replication_connection.ex

```diff
@@ -118,9 +118,9 @@ defmodule Realtime.Tenants.ReplicationConnection do
         socket_options: [ip_version],
         backoff_type: :stop,
         sync_connect: true,
-        parameters: [
-          application_name: connection_opts.application_name
```

Comment:
This doesn't work?

Reply:
from what I understood no, it's better to set after connection

Reply:
Unless it errors and never sets the application_name.  

We can see the application_name in the connection logs in Postgres, and that is a lot of times helpful to know if some application is reconnecting a lot.

Reply:
I will need to recheck to see if this works but I don't think it does 🤔

Reply:
You can check the Postgres logs and see it's working:

`connection authorized: user=supabase_admin database=postgres application_name=realtime_rls SSL enabled (protocol=TLSv1.3, cipher=TLS_AES_256_GCM_SHA384, bits=256)`

Reply:
At least with our other connections...

Reply:
will add back and remove this change